AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -1151,6 +1170,21 @@ - (void)testSessionTrackerStarted {
   XCTAssertTrue([MSAnalytics sharedInstance].sessionTracker.started);
 }
 
+- (void)testSessionTrackerStartedWithToken {
+
+  // When
+  [MSAppCenter startFromLibraryWithServices:@ [[MSAnalytics class]]];
```

Comment:
Did you run the formatter? Not sure if this is a typo or a weird side effect of the formatter.

Reply:
@achocron I did. I just copied the pattern from the test above this one.